src/blob_file_cache.h

```diff
@@ -22,7 +22,12 @@ class BlobFileCache {
   // bytes. The provided buffer is used to store the record data, so
   // the buffer must be valid when the record is used.
   Status Get(const ReadOptions& options, uint64_t file_number,
-             const BlobHandle& handle, BlobRecord* record, OwnedSlice* buffer);
+             const BlobHandle& handle, BlobRecord* record, OwnedSlice* buffer) {
+    return Get(options, file_number, handle, record, buffer, false);
+  }
+  Status Get(const ReadOptions& options, uint64_t file_number,
```

Comment:
How about just setting a default value for `for_compaction`

Reply:
Done